test/common/models/Message.js

```diff
@@ -51,10 +51,10 @@ describe('The Message class', function() {
       expect(message.hasAttachment).to.equal(false);
       expect(message.headers).to.deep.equal({});
       expect(message.from).to.deep.equal(jmap.EMailer.unknown());
-      expect(message.to).to.deep.equal([jmap.EMailer.unknown()]);
-      expect(message.cc).to.deep.equal([jmap.EMailer.unknown()]);
-      expect(message.bcc).to.deep.equal([jmap.EMailer.unknown()]);
-      expect(message.replyTo).to.deep.equal(jmap.EMailer.unknown());
+      expect(message.to).to.deep.equal([]);
+      expect(message.cc).to.deep.equal([]);
+      expect(message.bcc).to.deep.equal([]);
+      expect(message.replyTo).to.deep.equal([]);
```

Comment:
can we have a JSON object called expectedMessage for example with which we can perform deepEqual test?